ads/google/a4a/test/test-utils.js

```diff
@@ -161,15 +161,23 @@ describe('Google A4A utils', () => {
 
         url = ['https://foo.com?hello=world', 'https://bar.com?a=b'];
         const config = extractAmpAnalyticsConfig(a4a, headers);
-        const csiRequest = config.requests.visibilityCsi;
-        expect(csiRequest).to.not.be.null;
+        const iniLoadCsiRequest = config.requests.iniLoadCsi;
+        const renderStartCsiRequest = config.requests.renderStartCsi;
+        expect(iniLoadCsiRequest).to.not.be.null;
+        expect(renderStartCsiRequest).to.not.be.null;
         // We expect slotId == null, since no real element is created, and so
         // no slot index is ever set.
-        expect(csiRequest).to.match(new RegExp(
+        expect(iniLoadCsiRequest).to.match(new RegExp(
```

Comment:
P.S.  Looks like `expectMatchesAll` is defined in `test-performance.js`, so you'll have to factor it out into a utils file in this dir.  I think that if you don't put `test` in the name of that file, the test rig won't try to auto-run it.